src/screens/UpdateIconColor.tsx

```diff
@@ -89,9 +103,9 @@ const UpdateIconColor: React.FC = () => {
             width: "85%",
```

Comment:
Missed style, please remove and replace this embedded style with a style attribute!

Reply:
Resolved in f7e1296